product_profile/i18n/fr.po

```diff
@@ -0,0 +1,140 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* product_profile
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 8.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-06-23 07:14+0000\n"
+"PO-Revision-Date: 2015-06-23 09:57+0100\n"
+"Last-Translator: David BEAL <david.beal@akretion.com>\n"
+"Language-Team: Akretion\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: fr\n"
+"X-Generator: Poedit 1.7.5\n"
+
+#. module: product_profile
+#: help:product.profile,sequence:0
+msgid ""
+"Allows to define the order of the entries of profile_id field\n"
+"(not synchronized with product.template fields)"
+msgstr ""
+"Permet de définir l'ordre des entrées du champ profile_id\n"
+"(pas synchronisé avec les champs de product.template)"
+
+#. module: product_profile
+#: help:product.profile,explanation:0
+msgid ""
+"Allows to display an explanation on the selected profile\n"
+"(not synchronized with product.template fields)"
+msgstr ""
+"Permet d'afficher une explication sur le profil sélectionné.\n"
+"(pas synchronisé avec les champs de product.template)"
+
+#. module: product_profile
+#: field:product.profile,create_uid:0
+msgid "Created by"
+msgstr "Créé par"
+
+#. module: product_profile
+#: field:product.profile,create_date:0
+msgid "Created on"
+msgstr "Créé sur"
+
+#. module: product_profile
+#: field:base.config.settings,group_product_profile:0
+msgid "Display Profile fields"
+msgstr "Afficher les champs de Profile"
+
+#. module: product_profile
+#: help:base.config.settings,group_product_profile:0
+msgid ""
+"Display fields computed by product profile module.\n"
+"For debugging purpose\n"
+"see menu Sales > Configuration \n"
+"> Product Categories and Attributes\n"
+"> Product Profiles"
+msgstr ""
+"Afficher les champs calculés par le module Product Profile.\n"
+"A des fins de débuggage purpose\n"
+"voir menu Ventes > Configuration \n"
+"> Catégories de Produits et Attributs\n"
+"> Profils de Produit"
+
+#. module: product_profile
+#: field:product.profile,explanation:0
+msgid "Explanation"
+msgstr "Explication"
+
+#. module: product_profile
+#: field:product.mixin.profile,id:0 field:product.profile,id:0
+msgid "ID"
+msgstr "ID"
+
+#. module: product_profile
+#: field:product.profile,write_uid:0
+msgid "Last Updated by"
+msgstr "Dernière mise à jour par"
+
+#. module: product_profile
+#: field:product.profile,write_date:0
+msgid "Last Updated on"
```

Comment:
Cannot find this key in the code.
Anyway translation better like: "Dernière mise à jour le"